MonoGame.Framework/Graphics/GraphicsAdapter.cs

```diff
@@ -104,6 +104,9 @@ public DisplayMode CurrentDisplayMode
 #elif (WINDOWS && OPENGL) || LINUX
 
                 return new DisplayMode(OpenTK.DisplayDevice.Default.Width, OpenTK.DisplayDevice.Default.Height, (int)OpenTK.DisplayDevice.Default.RefreshRate, SurfaceFormat.Color);
+#elif WINDOWS
+                var dc = System.Drawing.Graphics.FromHwnd(IntPtr.Zero).GetHdc();
+                return new DisplayMode(GetDeviceCaps(dc, HORZRES), GetDeviceCaps(dc, VERTRES), GetDeviceCaps(dc, VREFRESH), SurfaceFormat.Color);

```

Comment:
Looks like this code needs to take into consideration the desktop zoom settings.  See https://github.com/mono/MonoGame/issues/2619#issuecomment-174216171